src/main/java/seedu/address/model/interview/UniqueInterviewList.java

```diff
@@ -70,6 +71,34 @@ public void setInterview(Interview target, Interview editedInterview) throws Dup
         internalList.set(index, editedInterview);
     }
 
+    /**
+     * Sorts the interviews by date and time.
+     */
+    public void sortInterviewsByDate() {
+        Comparator<Interview> dateSorter = (interview1, interview2) -> {
+            if (interview1.getDate().isBefore(interview2.getDate())) {
+                return -1;
+            } else if (interview1.getDate().isEqual(interview2.getDate())) {
+                if (interview1.getStartTime().isBefore(interview2.getStartTime())) {
+                    return -1;
+                } else if (interview1.getStartTime().isAfter(interview2.getStartTime())) {
+                    return 1;
+                } else if (interview1.getStartTime().equals(interview2.getStartTime())) {
+                    if (interview1.getEndTime().isAfter(interview2.getEndTime())) {
+                        return 1;
+                    } else {
+                        return 0;
+                    }
+                } else {
+                    return 0;
+                }
+            } else {
+                return 1;
+            }
+        };
```

Comment:
Perhaps can write a comment explaining briefly what -1, 0 and 1 is so save a step for the reader to visit the `Comparator` documentation to find out how the -1, 0, 1 affects the sorted ordr